install-dropday.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Name of the Magento container
+CONTAINER_NAME="magento-magento-1"
+
+# Ask for the username and password 
+read -p "Enter Dropday Username: " USERNAME
+read -s -p "Enter Dropday Password: " PASSWORD
+echo
+
+# Enter the Magento Docker container
+docker exec -it $CONTAINER_NAME bash -c "
+# Set up authentication for Composer
+composer global config http-basic.repo.magento.com \"$USERNAME\" \"$PASSWORD\"
+
+# Navigate to the Magento directory
+cd /bitnami/magento/
+
+# Use composer to install the Dropday extension
+composer require dropday-io/module-orderautomation
```

Comment:
Installing this extension with compose makes a copy of the master/main branch of this extension. While we want this Docker project to [enable developers to work on](https://github.com/dropday-io/Magento/issues/14) their current branch.
Therefor, the code should be mounted through the volumes. See code of this #15.

Reply:
the bash script created to install extension in the magento container because we need to install magento first in the container that was done on installing php apache magento and running all services there through a single image which is taken from public repo docker hub of bitnami. We can achieve this installation by creating docker file other then script but it will also doing the installtion through the script. The container have attached volume this extension will be stored in the volume this is the one time script running